packages/go/index.ts

```diff
@@ -479,13 +482,24 @@ function isReadable(v: any): v is Readable {
   return v && v.readable === true;
 }
 
-async function copyEntrypoint(entrypoint: string, dest: string): Promise<void> {
-  const data = await readFile(entrypoint, 'utf8');
+async function copyEntrypoint(files: string[], dest: string): Promise<void> {
+  for (const file of files) {
+    if (!existsSync(file)) {
```

Comment:
Why would the file not exist?

Reply:
This may have something to do with `analyze.go`, when the directory structure is like this:
```
.
├── README.md
├── api
│   ├── date.go
│   ├── hello.go
│   ├── json.go
│   └── myinfo.go
├── go.mod
└── util
    └── util.go
```
`analyzed.watch` in  the result is like this:

```
[ 'json.go', 'api/hello.go',  'util/util.go']
```

When copying the entry and watch files, I had to ignore `json.go` which does not exist relative to `goModAbsPathDir`.

Since `analyze.go` is used by both build and dev, it works fine when building for production, so I didn't modify `analyze.go` here. Instead, I added a check for the existence of the file when copying.

Reply:
Hmm that seems like a bug if it returned json.go instead of api/json.go

Perhaps the working directory is wrong?

Reply:
@styfle  I changed the `analyze.go` code a bit, so that the `entrypoint` path returned in `analyzed.watch` array works.

Also, I changed `/go/test/fixtures/08-include-files/another.go`, so that the `yarn test-integration-once` does not report any errors.